de.metas.business/src/main/java/org/adempiere/mm/attributes/countryattribute/ICountryAttributeDAO.java

```diff
@@ -44,16 +47,15 @@
 	AttributeId retrieveCountryAttributeId(final int adClientId, final int adOrgId);
 
 	/**
-	 * Gets the attribute value related to a country. The country attribute is set in Sysconfig de.metas.swat.CountryAttribute. If we don't find an attribute value, behavior is described by Sysconfig
+	 * * Gets the attribute value related to a country. The country attribute is set in Sysconfig de.metas.swat.CountryAttribute. If we don't find an attribute value, behavior is described by Sysconfig
 	 * de.metas.swat.CountryAttributeAction.
 	 * 
-	 * Part of task 05839.
-	 * 
 	 * @param ctx
```

Comment:
These javadoc-params were added by eclipse at a certain time, but they don't provide any useful information.
Generally, please fix or remove javadocs that are useless or lying when you come accross it (=>boyscout-rule)